Index: src/main/resources/data.sql
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+> delete from transaction;\n delete from expense;\n delete from processed_file;\nselect sysdate from dual;
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/data.sql b/src/main/resources/data.sql
--- a/src/main/resources/data.sql	
+++ b/src/main/resources/data.sql	
@@ -1,4 +1,4 @@
- delete from transaction;
- delete from expense;
- delete from processed_file;
+-- delete from transaction;
+-- delete from expense;
+-- delete from processed_file;
 select sysdate from dual;
\ No newline at end of file
